MovieApp/MovieDetailsViewController+Extensions.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  MovieDetailsViewController+Extensions.swift
+//  MovieApp
+//
+//  Created by endava-bootcamp on 05.04.2023..
+//
+
+import Foundation
+import MovieAppData
+import PureLayout
+import UIKit
+
+extension MovieDetailsViewController: UICollectionViewDataSource {
```

Comment:
I guess I was not clear on this one. Extensions should have a dedicated file/folder. But these kind of extensions, protocol conformance, like in this example, should be in the same class file. You just add the extension to organize the code better and to make it more readable. Something like this, in the same file. Where the collection view delegate methods are in that extension.

```
class MovieDetailsViewController: ViewController { 

}

extension MovieDetailsViewController: UICollectionViewDelegate { 

}
```